test/NCalc.Tests/Fixtures.cs

```diff
@@ -225,6 +225,7 @@ public void Maths()
             Assert.Equal(1d, Extensions.CreateExpression("Floor(1.5)").Evaluate());
             Assert.Equal(-1d, Extensions.CreateExpression("IEEERemainder(3,2)").Evaluate());
             Assert.Equal(0d, Extensions.CreateExpression("Log(1,10)").Evaluate());
+            Assert.Equal(0d, Extensions.CreateExpression("Ln(1)").Evaluate());

```

Comment:
I know you're piggy-backing on the existing test cases, but 1 probably isn't the best value to take the log of for any of these since it isn't affected by the base.

Reply:
I'll add more tests but in fact we don't need to test dotnet built-in classes

Reply:
I wasn't asking for additional tests.  I'm saying take the log of a number other than 1 for the test because Log(1,10) and Ln(1) are both 0 which doesn't tell you that the correct function is being called.  

```
Assert.Equal(1d, Extensions.CreateExpression("Log(10,10)").Evaluate());
Assert.Equal(2.30258509299405, Extensions.CreateExpression("Ln(10)").Evaluate());
Assert.Equal(1d, Extensions.CreateExpression("Log10(10)").Evaluate());
```